src/main/java/saasquatch/extintegration/EIHeaders.java

```diff
@@ -0,0 +1,30 @@
+package saasquatch.extintegration;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import javax.annotation.Nonnull;
+import com.google.common.net.HttpHeaders;
+
+public class EIHeaders {
+
+  public static Map<String, String> getGlobalHeaders(@Nonnull Collection<String> frameSrc) {
+    final List<String> csp = new ArrayList<>();
+    csp.add("default-src https: 'unafe-inline' 'unsafe-eval'");
+    if (!frameSrc.isEmpty()) {
+      csp.add("frame-src " + String.join(" ", frameSrc));
+    }
+    final Map<String, String> headers = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+    headers.put(HttpHeaders.STRICT_TRANSPORT_SECURITY, "max-age=31536000");
+    headers.put(HttpHeaders.CONTENT_SECURITY_POLICY, String.join("; ", csp));
+    headers.put(HttpHeaders.X_FRAME_OPTIONS, "SAMEORIGIN");
```

Comment:
Scott have you tested that the Segment or Stripe integration still works with this? I don't the portal will be able to load them with this config.